base/src/main/java/com/smartdevicelink/proxy/rpc/GetVehicleDataResponse.java

```diff
@@ -1,597 +1,649 @@
+        return this;
+    }
+
+    public Integer getRpm() {
+        return getInteger(KEY_RPM);
+    }
+
+    /**
+     * Sets Fuel Level State
+     *
+     * @param fuelLevelState a ComponentVolumeStatus related to FuelLevel State
+     */
+    @Deprecated
+    public GetVehicleDataResponse setFuelLevelState(ComponentVolumeStatus fuelLevelState) {
+        setParameters(KEY_FUEL_LEVEL_STATE, fuelLevelState);
+        return this;
+    }
+
+    /**
+     * Gets Fuel Level State
+     *
+     * @return a ComponentVolumeStatus related to FuelLevel State
+     */
+    @Deprecated
+    public ComponentVolumeStatus getFuelLevelState() {
+        return (ComponentVolumeStatus) getObject(ComponentVolumeStatus.class, KEY_FUEL_LEVEL_STATE);
+    }
+
+    /**
+     * Sets the fuelLevel.
+     *
+     * @param fuelLevel The fuel level in the tank (percentage). This parameter is deprecated starting RPC Spec
+     *                  7.0, please see fuelRange.
+     * @since SmartDeviceLink 7.0.0
+     */
+    @Deprecated
+    public GetVehicleDataResponse setFuelLevel(Double fuelLevel) {
+        setParameters(KEY_FUEL_LEVEL, fuelLevel);
+        return this;
+    }
+
+    /**
+     * Gets the fuelLevel.
+     *
+     * @return Double The fuel level in the tank (percentage). This parameter is deprecated starting RPC Spec
+     * 7.0, please see fuelRange.
+     * @since SmartDeviceLink 7.0.0
+     */
+    @Deprecated
+    public Double getFuelLevel() {
+        Object object = getParameters(KEY_FUEL_LEVEL);
+        return SdlDataTypeConverter.objectToDouble(object);
+    }
+
+    public GetVehicleDataResponse setInstantFuelConsumption(Double instantFuelConsumption) {
+        setParameters(KEY_INSTANT_FUEL_CONSUMPTION, instantFuelConsumption);
+        return this;
+    }
+
+    public Double getInstantFuelConsumption() {
+        Object object = getParameters(KEY_INSTANT_FUEL_CONSUMPTION);
+        return SdlDataTypeConverter.objectToDouble(object);
+    }
+
+    /**
+     * Sets the externalTemperature.
+     *
+     * @param externalTemperature The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * {"num_min_value": -40.0, "num_max_value": 100.0}
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
+     */
+    @Deprecated
+    public GetVehicleDataResponse setExternalTemperature(Double externalTemperature) {
+        setParameters(KEY_EXTERNAL_TEMPERATURE, externalTemperature);
+        return this;
+    }
+
+    /**
+     * Gets the externalTemperature.
+     *
+     * @return Float The external temperature in degrees celsius. This parameter is deprecated starting RPC
+     * Spec 7.1, please see climateData.
+     * {"num_min_value": -40.0, "num_max_value": 100.0}
+     * @since SmartDeviceLink 2.0.0
+     * @deprecated in SmartDeviceLink 7.1.0
+     */
+    @Deprecated
+    public Double getExternalTemperature() {
```

Comment:
@kostyaBoss When using the RPC generator tool, on line 243 I got `Float` instead of `Double`. I would replace `Double` with `Float`.